python/GafferUI/GLWidget.py

```diff
@@ -172,7 +175,10 @@ def _makeCurrent( self ) :
 			return False
 
 		self._qtWidget().viewport().makeCurrent()
-		return True
+
+		# We've set the context, but if we haven't called IECoreGL.init yet,
```

Comment:
What happens if we actually call `IECoreGL.init()` here? Seems like that might be preferable if it works.